tests/entropy_initial_lookup.prm

```diff
@@ -0,0 +1,152 @@
+  # The 'ascii data' option does not need the additional
+  # plugin, but is limited to the available adiabat data files.
+  # 'compute entropy profile' computes arbitrary adiabats
+  # internally, based on the data table.
+
+  set Model name = compute entropy profile
+
+  subsection Compute entropy profile
+    set Surface entropy = 2534  # equivalent to T=1600 K
+  end
+
+  subsection Ascii data model
+    set Data directory  = $ASPECT_SOURCE_DIR/data/material-model/
+    set Data file name  = entropy-table/opxtable/1600K_adiabat.txt
+  end
+end
+
+
+# We prescribe temperatures according to the data table.
+# This output is computed in the material model.
+subsection Temperature field
+  set Temperature method = prescribed field with diffusion
+end
+
+
+# We solve the entropy equation for the compositional field with name
+# 'entropy'. Temperature and density are then computed based on entropy and
+# pressure.
+subsection Compositional fields
+  set Number of fields = 2
+  set Names of fields = entropy, density_field
+  set Types of fields = entropy, density
+  set Compositional field methods = field, prescribed field
+end
+
+
+subsection Boundary temperature model
+  set Fixed temperature boundary indicators = top
+  set List of model names = initial temperature
+end
+
+
+# Prescribing downward flow through a vertical pipe with
+# tangential side boundaries.
+subsection Boundary velocity model
+  set Prescribed velocity boundary indicators = top:function
+  set Tangential velocity boundary indicators = left, right
+  subsection Function
+    set Function expression = 0; -0.01
+  end
+end
+
+
+subsection Gravity model
+  set Model name = vertical
+
+  subsection Vertical
+    set Magnitude = 9.81
+  end
+end
+
+
+# We look up the values of entropies from another look-up table.
+# This table looks up the value of entropy from a pair of temperature and pressure.
+subsection Initial composition model
+  set List of model names = entropy table lookup
+  subsection Entropy table lookup
+    set Data directory = $ASPECT_SOURCE_DIR/data/material-model/entropy-table/pyrtable/
+    set Material file name = material_table_temperature_pressure.txt
+  end
+end
+
+
+# We use a data table for orthopyroxene computed using the thermodynamic
+# modeling software Perple_X.
+subsection Material model
+  set Model name = entropy model
+
+  subsection Entropy model
+    set Data directory                   = $ASPECT_SOURCE_DIR/data/material-model/entropy-table/pyrtable/
+    set Material file name               = material_table_entropy_pressure.txt
+    set Lateral viscosity file name      = constant_lateral_vis_prefactor.txt
+    set Thermal conductivity formulation = constant
+    set Thermal conductivity             = 0.1
+  end
+
+  subsection Depth dependent viscosity
+    set Data directory                   = $ASPECT_SOURCE_DIR/benchmarks/entropy_adiabat/data/
+    set Data file name                   = constant_viscosity.txt
+  end
```

Comment:
I think this part is not used, correct?

Reply:
Yes, this part is not needed. I have removed them from the two tests and remade the outputs.